Source/Relativity.Test.Helpers/Relativity.Test.Helpers/TestHelper.cs

```diff
@@ -137,7 +154,50 @@ public IStringSanitizer GetStringSanitizer(int workspaceID)
 		{
 			throw new NotImplementedException();
 		}
-	}
-
 
+		private void InstallKeplerResourceFiles(List<string> keplerFiles)
+		{
+			foreach (var keplerDllName in keplerFiles)
+			{
+				using (IRSAPIClient rsapiClient = GetServiceFactory().CreateProxy<IRSAPIClient>())
+				{
+					var rfRequest = new ResourceFileRequest
+					{
+						AppGuid = new Guid(_defaultAppGuid),
+						FullFilePath = Path.Combine(_keplerFileLocation, keplerDllName),
+						FileName = keplerDllName
+					};
+					try
+					{
+						rsapiClient.PushResourceFiles(rsapiClient.APIOptions, new List<ResourceFileRequest>() { rfRequest });
+						Console.WriteLine($"{nameof(InstallKeplerResourceFiles)} - File ({keplerDllName}) - was uploaded successfully");
+					}
+					catch (Exception ex)
+					{
+						Console.WriteLine($"{nameof(InstallKeplerResourceFiles)} - Could not upload ({keplerDllName}) - Exception: {ex.Message}");

```

Comment:
throw exception. we are eating the exception here.

Reply:
fixed

Reply:
Why are we adding kepler services to an environment?

Reply:
@dvbarnes This is part of a bigger change we are doing to TestHelpers where we are replacing methods that use DbContext with Kepler Services

Reply:
Do you have a why behind this? I see this as more bloat instead of adding benefit